meetings/2024-11-13.md

```diff
@@ -0,0 +1,109 @@
+
+### Meeting Guidelines
+* [W3C Solid Community Group Calendar](https://www.w3.org/groups/cg/solid/calendar).
+* [W3C Solid Community Group Meeting Guidelines](https://github.com/w3c-cg/solid/blob/main/meetings/README.md).
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Conduct](https://www.w3.org/policies/code-of-conduct/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* elf Pavlik
+
+### Introductions
+*
+
+---
+
+## Topics
+
+### Relation between what is sent and what is received
+* MdJ: Question about the spec text: https://github.com/solid/specification/issues/693
+* MdJ: I noticed that the solid spec talks about storage but it only describes the interface. It doesn't say how the state of the resource should be kept over time. Rahul answered that i follows from HTTP semantics.
+* eP: Did you stumble into a specific problem?
+* MdJ: The preserving of turtle syntax, I wasn't sure if that was a requirement
+* eP: To my understanding one can't rely on specific formatting be preserved
+* PAC: I don't think that HTTP semantics guarantees that the conten has to be the same or not. For PUT, the server is allowed to make changes to the content you put. Comments being stripped from turtle could be an issue. I think spec should say something about reliance on what can be PUT and GET. LDP says what containment triples can be added. We should look into it.
+* TBL: There was a question on the forum about having a solid pod on top of SPARQL store. I'm trying not to put data in comments but actual `rdfs:comment` statements. Maybe in the storage metadata we can say if this server is preserving formatting or not.
+* MdJ: What if you have a cluster with multiple masters and later the nodes talk to each other and something else happens.
+* RG: The data that you GET or PUT is always a representation, how it maps to the state is a perogative of the server. We can decide if we want it, the server may say that it will store the turtle format or not. If you do it on the spec or implementation level is missing from the implementation.
```

Comment:
```suggestion
* RG: The data that you `GET` or `PUT` is always a representation. How it maps to the state is a prerogative of the server. We can decide if we want it. The server may say whether or not it will store the Turtle format. Whether you do it on the spec or implementation level is missing from the implementation.
```